Purchases/Purchasing/RCStoreKitWrapper.m

```diff
@@ -66,6 +66,14 @@ - (void)finishTransaction:(SKPaymentTransaction *)transaction
     [self.paymentQueue finishTransaction:transaction];
 }
 
+- (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0)) API_UNAVAILABLE(tvos, macos, watchos) {
+    if (@available(iOS 14.0, *)) {
+        [self.paymentQueue presentCodeRedemptionSheet];
```

Comment:
this gives an error 

`No visible @interface for 'SKPaymentQueue' declares the selector 'presentCodeRedemptionSheet'`

on latest version 3.9.0.

Reply:
Which Xcode version are you running? 

This method is only available when compiling on Xcode versions >= 12.0. 
@alicanbatur Do you have any specific needs that force you to use earlier Xcode versions?